data/base/shaders/skybox.vert

```diff
@@ -0,0 +1,41 @@
+// Version directive is set by Warzone when loading the shader
+// (This shader supports GLSL 1.20 - 1.50 core.)
+
+uniform mat4 posMatrix;
+
+uniform int fog_enabled;
+uniform vec4 fog_color;
```

Comment:
I'm just curious: why the `snake_case`? It's a little bit inconsistent with other variable names.
(Same in the `skybox.frag`.)

Reply:
Yea inconsistency is key 😂 jokes aside, I took the names from another place. I dont wanna make a huge refactor when implementing smn.